app/recordtransfer/models.py

```diff
@@ -407,6 +408,42 @@ def copy_session_uploads(
 
         return copied, missing
 
+    def get_quantity_and_unit_of_measure(self) -> str:
+        """Create a human-readable statement of how many files are in this session.
+
+        If the session is in the state UPLOADING, returns a count of temp files. If the session is
+        in the state STORED, returns a count of permanent files. If the session is in the state CREATED,
+        returns an appropriate value that indicates the lack of files.
+
+        Uses the :ref:`ACCEPTED_FILE_FORMATS` setting to group file types together.
+
+        Returns:
+            A human readable count and size of all files in this session.
+        """
+        if not settings.FILE_UPLOAD_ENABLED:
+            return
+
+        if self.status not in (
+            self.SessionStatus.CREATED,
+            self.SessionStatus.UPLOADING,
+            self.SessionStatus.STORED,
+        ):
+            raise ValueError(
+                f"Cannot get quantity and unit of measure from session {self.token} because the "
+                f"session status is {self.status} and not {self.SessionStatus.UPLOADING} or "
+                f"{self.SessionStatus.STORED}"
+            )
+
+        size = get_human_readable_size(self.upload_size, base=1024, precision=2)
```

Comment:
Change based on #378

```suggestion
        size = get_human_readable_size(self.upload_size, base=1000, precision=2)
```

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/381/commits/dd01e0df871b4f46d424e5f5b2fb84e120774bd5